pulpcore/cli/common/__init__.py

```diff
@@ -92,6 +93,17 @@ def _debug_callback(level: int, x: str) -> None:
     ctx.obj = PulpContext(api_kwargs=api_kwargs, format=format, background_tasks=background)
 
 
+@main.command()
+@click.option("--name", required=True)
+@click.option("--min-version")
+@click.option("--max-version")
+@pass_pulp_context
+def has_plugin(pulp_ctx: PulpContext, name: str, min_version: str, max_version: str) -> None:
```

Comment:
@daviddavis this is an alternative approach to query for versions in the tests.

Reply:
I use the api json schema a lot (since our help texts are rather incomplete) and was thinking of adding a command to return the api schema json. I wonder if maybe we should have a `debug` group for these commands? wdyt?

Reply:
love the debug idea.
We can include jq like postprocessing into the schema json. :thinking: or into all the outputs...

Reply:
Cool, I'll approve this PR if you just want to merge as is.

Reply:
added "debug", and thrilled to see more helper there.